app/res/layout/entity_detail_list.xml

```diff
@@ -4,9 +4,18 @@
               android:layout_height="match_parent"
               android:orientation="vertical"
     >
+
+    <LinearLayout
+        android:id="@+id/entity_detail_header"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_above="@+id/screen_entity_detail_list"
+        android:orientation="horizontal">

```

Comment:
It matters a bit due to the way the stuff gets displayed in views, but possibly not enough that we should worry about it